### ISSUE TITLE
mapValue does not work with allowMultiple

### ISSUE THREAD

**Issue by ryan-slack-big** [aml-org/amf#859]
Your issue may already be reported! Please search on Github issues before creating one.

* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] question
  - [ ] other

* **What is the current behavior?**

I would like to validate this:
```
#%Map List Test 1.0
codes:
  "FF0000":
      - red
      - RED
      - Red
      - rouge
```
I would expect the following dialect to work:
```
#%Dialect 1.0

dialect: Map List Test
version: 1.0

documents:
  root:
    encodes: MapListNode

nodeMappings:
  MapListNode:
    mapping:
      codes:
        range: CodeNames
        mapKey: code
        mapValue: names

  CodeNames:
    mapping:
      code:
        range: string
        mandatory: true
        unique: true
      names:
        range: string
        mandatory: true
        allowMultiple: true
```
however this results in
```
- Source: http://a.ml/vocabularies/amf/core#syaml-error
  Message: Expected scalar but found: [red, RED, Red, rouge]
  Level: Violation
  Target: 
  Property: 
  Position: Some(LexicalInformation([(4,11)-(9,0)]))
  Location: file://fiddle/maplist-test.yaml
```

On the other hand this works...
```
#%Map List Test 1.0

codes:
  "FF0000":
    names:
      - red
      - RED
      - Red
      - rouge
```
...with this dialect
```
#%Dialect 1.0

dialect: Map List Test
version: 1.0

documents:
  root:
    encodes: MapListNode

nodeMappings:
  MapListNode:
    mapping:
      codes:
        range: CodeNames
        mapKey: code

  CodeNames:
    mapping:
      code:
        range: string
        mandatory: true
        unique: true
      names:
        range: string
        mandatory: true
        allowMultiple: true
```

*Environment:*
  - AMF Version: 4.7.1 (4a8083c28)
  - AMF Distribution: JVM
  - Operating System: Liinux

**Comment by looseale** [comment]
Hi @ryan-slack-big . I could replicate this. I created a bug in our internal issue system to track this: https://www.mulesoft.org/jira/browse/APIMF-2955. I wrote this here to link our internal ticket with Github.

All updates on this issue will be communicated through this channel (the Github issue).
Thanks!

**Comment by tomsfernandez** [comment]
Hi @ryan-slack-big. This issue is fixed with #271 . It will be included in the next release that will be done sometime next week. I'm closing this issue.
